packages/react-native/example/package.json

```
@@ -7,25 +7,41 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint .",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "postinstall": "cd node_modules/@notifee/react-native && yarn"
   },
   "dependencies": {
-    "@notifee/react-native": "link:../",
-    "react": "17.0.2",
-    "react-native": "0.66.0"
+    "react": "18.1.0",
+    "react-native": "0.70.4",
+    "@notifee/react-native": "file:../"
```

I think this, to match prior?

```suggestion
    "@notifee/react-native": "link:../"
```

---

this wasn't working for me so i changed it to "file...".

---

works for me - if you have direct testing experiment I'm 100% with that, I was just scanning for differences and things I know of from working with react-native releases group